project/api/views/endpoints/stv_downloader.py

```diff
@@ -66,10 +59,39 @@ def stv_downloader(request, primary_key):
 
     end_date = jd2gcal(stv[0].end_date, 0)
     end_string = "{}-{}-{}".format(end_date[0], end_date[1], end_date[2])
+
     response = JsonResponse(geojson)
 
     response["Content-Disposition"] = "attachment;filename={}_{}_{}.json;".format(
         stv[0].entity.label, start_string, end_string
     )
-
     return response
+
+def stv_downloader(request, primary_key):
+    """
+    Download stvs as geojson.
+    """
+
+    stv = SpacetimeVolume.objects.filter(pk=primary_key)
+    if len(stv) == 0:
+        return HttpResponse(status=404)
+
+    return stv_to_geojson_response(stv)
+
+
+def historical_stv_downloader(request, primary_key):
+    """
+    Download historical stvs as geojson.
+    """
+
+    try:
+        history = HistoricalSpacetimeVolume.objects.get(history_id=primary_key)
+    except HistoricalSpacetimeVolume.DoesNotExist:
+        return HttpResponse(status=404)
+
+    stv = SpacetimeVolume.objects.filter(id=history.id)
```

Comment:
If I am reading this right - you are returning an element from SpacetimeVolume (currently available STVs)
I thought, that `history` object will contain geometry and all other fields that were requested.

Reply:
Yes, it returns as a FeatureCollection that contains the instance's other fields as attributes as well

Reply:
What do you return then? Why do you searching for STV when you've already have the history object?

What if STV was deleted or changed, then you'll have the incorrect (newer) geometry